src/components/ProductCard.js

```diff
@@ -24,37 +25,33 @@ const Image = styled(Img)`
   }
 `
 
-const Card = styled(Link)`
+const Card = styled.div`
   color: ${(props) => props.theme.colors.text};
-  box-shadow: 0px 14px 66px rgba(0, 0, 0, 0.07),
-    0px 10px 17px rgba(0, 0, 0, 0.03), 0px 4px 7px rgba(0, 0, 0, 0.05);
-
+  box-shadow: 0px 14px 66px rgba(0, 0, 0, 0.07);
   display: flex;
   flex-direction: column;
   justify-content: space-between;
   background: ${(props) => props.theme.colors.searchBackground};
   border-radius: 4px;
   border: 1px solid ${(props) => props.theme.colors.lightBorder};
   text-decoration: none;
-
   &:hover {
-    box-shadow: 0px 8px 17px rgba(0, 0, 0, 0.15);
-    background: ${(props) => props.theme.colors.tableBackgroundHover};
     transition: transform 0.1s;
     transform: scale(1.02);
   }
 `
 
 const Content = styled.div`
   padding: 1.5rem;
+  padding-right: 1rem;
```

Comment:
```suggestion
```
I don't see any reason to have different `padding-right`

Reply:
That was in lieu of a slight negative margin on the github component so it's right-sided alignment is in-line with the right-side of the bottom button